libs/coin-modules/coin-icon/src/js-buildTransaction.ts

```diff
@@ -0,0 +1,53 @@
+import type { IconAccount, Transaction } from "./types";
+import IconService from "icon-sdk-js";
+import type { IcxTransaction } from "icon-sdk-js";
+import { getNid, getNonce } from "./logic";
+import BigNumber from "bignumber.js";
+import { RPC_VERSION } from "./constants";
+
+const { IconBuilder, IconConverter } = IconService;
+
+const buildTransferTransaction = (
+  a: IconAccount,
+  t: Transaction,
```

Comment:
```suggestion
  account: IconAccount,
  transaction: Transaction,
```

Let's avoir 1 letter parameters